articles/quickstart/native/android-vnext/00-login.md

```diff
@@ -0,0 +1,250 @@
+
+Replace `YOUR_APP_PACKAGE_NAME` with your application's package name, available as the `applicationId` attribute in the `app/build.gradle` file.
+
+## Install the Auth0 Android SDK
+
+Add the [Auth0 Android](https://github.com/auth0/Auth0.Android) SDK into your project. The library will make requests to the Auth0's Authentication and Management APIs.
+
+### Add Auth0 to Gradle
+
+In your app's `build.gradle` dependencies section, add the following:
+
+```groovy
+apply plugin: 'com.android.application'
+android {
+  // ...
+}
+dependencies {
+  // Add the Auth0 Android SDK
+  implementation 'com.auth0.android:auth0:1.+'
+}
+```
+
+::: note
+If Android Studio lints the `+` sign, or if you want to use a fixed version, check for the latest in [Maven](http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22auth0%22%20g%3A%22com.auth0.android%22) or [JCenter](https://bintray.com/auth0/android/auth0).
+:::
+
+::: panel Sync Project with Gradle Files
+Remember to synchronize using the Android Studio prompt or run `./gradlew clean build` from the command line. For more information about Gradle usage, check [their official documentation](http://tools.android.com/tech-docs/new-build-system/user-guide).
+:::
+
+Add manifest placeholders required by the SDK. The placeholders are used internally to define an `intent-filter` that captures the authentication callback URL.
+
+To add the manifest placeholders, add the next line:
+
+```xml
+// app/build.gradle
+
+apply plugin: 'com.android.application'
+compileSdkVersion 30
+android {
+    defaultConfig {
+        applicationId "com.auth0.samples"
+        minSdkVersion 21
+        targetSdkVersion 30
+        // ...
+
+        // ---> Add the next line
+        manifestPlaceholders = [auth0Domain: "@string/com_auth0_domain", auth0Scheme: "demo"]
+        // <---
+    }
+}
+```
+
+::: note
+You do not need to declare a specific `intent-filter` for your activity, because you have defined the manifest placeholders with your Auth0 **Domain** and **Scheme** values and the library will handle the redirection for you.
+:::
+
+The `AndroidManifest.xml` file should look like this:
+
+```xml
+<!-- app/src/main/AndroidManifest.xml -->
+
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.auth0.samples">
+
+    <uses-permission android:name="android.permission.INTERNET" />
+
+    <application
+        android:allowBackup="true"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:theme="@style/AppTheme">
+
+        <activity android:name="com.auth0.samples.MainActivity">
+          <intent-filter>
+              <action android:name="android.intent.action.MAIN" />
+              <category android:name="android.intent.category.LAUNCHER" />
+          </intent-filter>
+        </activity>
+
+    </application>
+
+</manifest>
```

Comment:
@lbalmaceda Can you check this is all still relevant?

Reply:
The only line that matters here is the uses-permission, because the permission is required for the library to make network requests. I'd say the rest in a way gives context to locate the change. But you could also just bring up that line and say "add this permission to your application's manifest"